src/style/_type.scss

```diff
@@ -1,21 +1,42 @@
-@mixin text_body($color: $text-body-default-font-color)
+@mixin text_body($color: $regular-text-font-color,$weight: $regular-text-font-weight, $size: $regular-text-font-size)
 {
     font-family: sans-serif;
+    font-size: $size;
+    font-weight: $weight;
+    color: $color;
+}
 
+@mixin text_title($color: $title-text-font-color, $size: $title-text-font-size)
+{
+    font-family: $title-text-font-family;
+    font-size: $size;
+    font-weight: $title-text-font-weight;
     color: $color;
 }
 
 @mixin text_code($color: $text-code-default-font-color)
 {
-    font-family: monospace;
-    font-weight: 600;
-
-    color: $color;
+  font-family: $code-default-font-family;
```

Comment:
should be 4 space indentation.
I think you may need to change your setting is vscode